UnityMcpBridge/UnityMcpServer~/src/tools/manage_editor.py

```
@@ -63,4 +57,4 @@ def manage_editor(
             return response if isinstance(response, dict) else {"success": False, "message": str(response)}
 
         except Exception as e:
-            return {"success": False, "message": f"Python error managing editor: {str(e)}"}
\ No newline at end of file
+            return {"success": False, "message": f"Python error managing editor: {str(e)}"}
```

Ruff (0.13.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)